ui/src/views/infra/network/ServiceProvidersTab.vue

```diff
@@ -937,10 +937,38 @@ export default {
             }
           ]
         },
-        // {
-        //   title: 'SecurityGroupProvider',
-        //   details: ['name', 'state', 'id', 'servicelist'],
-        // },
+        {
+          title: 'SecurityGroupProvider',
```

Comment:
is it disabled by default in advanced zone ?
can you add a doc PR to tell how to support SG ? it seems users only need to enable the provider ?

can SG be supported per network or per account ?

Reply:
Let me create the docs for this PR